cmd/config/cache/config.go

```diff
@@ -38,6 +38,8 @@ type Config struct {
 	WatermarkLow  int      `json:"watermark_low"`
 	WatermarkHigh int      `json:"watermark_high"`
 	Range         bool     `json:"range"`
+	CommitAsync   bool     `json:"-"`
+	Commit        string   `json:"commit"`
```

Comment:
you don't need to remember this value `CommitAsync` false means we are in sync

Reply:
while marshaling the config for `mc admin config set/get` command, this field is required afaics.